status-client-spec.md

```diff
@@ -107,19 +107,8 @@ nodes allow you to discover other nodes of the network.
 Currently the main bootstrap nodes are provided by Status Gmbh, but anyone can
 run these provided they are connected to the rest of the Whisper network.
 
-Status maintains a list of boootstrap nodes in the following locations:
-- Asia:
```

Comment:
In line with above discussion, these should be fallback. I'd put them in an appendix and specify MAY or SHOULD specify fallback mechanism for bootstrap nodes in case DNS has issues, etc. It doesn't belong in main spec imo.